interface/forms/aftercare_plan/new.php

```diff
@@ -1,20 +1,22 @@
 <?php
 
-/**
+/*
  * aftercare_plan new.php
  *
  * @package   OpenEMR
  * @link      http://www.open-emr.org
  * @author    Naina Mohamed <naina@capminds.com>
  * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
  * @copyright Copyright (c) 2012-2013 Naina Mohamed <naina@capminds.com> CapMinds Technologies
  * @copyright Copyright (c) 2017-2019 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2022 Sherwin Gaddis <sherwingaddis@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 require_once(__DIR__ . "/../../globals.php");
-require_once("$srcdir/api.inc.php");
-require_once("$srcdir/patient.inc.php");
+require_once("$srcdir/api.inc");
+require_once("$srcdir/patient.inc");
```

Comment:
Are these bootstrapped to the .php version?

Reply:
need to revert this block back to .php endings